CHANGELOG.md

```diff
@@ -6,17 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Password changes (`PUT /authn/:account/password`) now produce audit events with
+  message ID `password` ([cyberark/conjur#1548](https://github.com/cyberark/conjur/issues/1548))
```

Comment:
What about POSTing or any other HTTPVERBing?  Did any of those already have events, or do we only support PUT?

Reply:
It's only `PUT`: https://github.com/cyberark/conjur/blob/db6126023a664f4b498c18e2686a3ca7ef00c650/config/routes.rb#L42